cpp/oneapi/dal/algo/polynomial_kernel/test/batch.cpp

```diff
@@ -72,18 +72,15 @@ class polynomial_kernel_batch_test : public te::algo_fixture {
                               const polynomial_kernel::compute_result<>& result) {
         const auto result_values = result.get_values();
 
-        SECTION("result values table shape is expected") {
-            REQUIRE(result_values.get_row_count() == x_data.get_row_count());
-            REQUIRE(result_values.get_column_count() == y_data.get_row_count());
-        }
-
-        SECTION("there is no NaN in result values table") {
-            REQUIRE(te::has_no_nans(result_values));
-        }
-
-        SECTION("result values are expected") {
-            check_result_values(scale, shift, degree, x_data, y_data, result_values);
-        }
+        INFO("result values table shape is expected")
```

Comment:
```suggestion
        INFO("check if result values table shape is expected")
```

This is wording used in other tests. The same for other `INFO`